blueocean-dashboard/src/main/js/components/Activity.jsx

```diff
@@ -88,23 +88,24 @@ export class Activity extends Component {
         }
 
         const latestRun = runs[0];
+        const head = 'pipelinedetail.activity.header';
```

Comment:
Yeah, I'll admit I am not sure which approach is the best. Part of me felt that using the full keys in all places would make it easier to search across the code base and see where certain keys were used without having to search for substrings of the keys. On the other hand, it is a little verbose and repetitive in the code. Curious what the the rest of the team feels about this, cc @imeredith @kzantow @tfennelly @sophistifunk @michaelneale